File: CP.Backend/API/Models/Event/Event.cs
using System;
using System.Collections.Generic;

namespace API.Models.Event
{
    public class Event
    {
        public Guid Id { get; set; }
        public Guid CreatorUserId { get; set; }
        public string Title { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public bool AllDay { get; set; }

        public virtual EventDetails Details { get; set; }
        public virtual ICollection<Attendee> Attendees { get; set; }
    }
}

File: CP.Frontend/src/models/Event.ts
export type Event = {
  id: string;
  creatorUserId: string;
  title: string;
  start: Date;
  end: Date;
  allDay: false;
  details: {
    description: string;
  };
  attendees: Array<{
    userId: string;
  }>;
};
